style.css

```diff
@@ -98,3 +102,165 @@ section ul .linked-in,
   width: 30px;
   height: 30px;
 }
+
+/* Styling the works section */
+
+.grid-container {
+  display: grid;
+}
+
+.works {
+  padding-right: 24px;
+  padding-left: 24px;
+  margin: auto 0;
+  padding-top: 114px;
+}
```

Comment:
- [ ] Please kindly align the contents of this container with `flexbox` property according to the requirement. 👍